app/models/reaction.rb

```diff
@@ -170,6 +170,8 @@ def permissions
   end
 
   def negative_reaction_from_untrusted_user?
+    return if user.any_admin?
```

Comment:
I assume all admins are implicitly trusted, let me know if I should remove this guard.